how-to/how-to-obfuscate-java-code-with-proguard.md

```diff
@@ -0,0 +1,96 @@
+---
+title: How to Obfuscate Java Code With ProGuard
+description: Obfuscating the Java Code of an AppBuilder Apache Cordova app
+type: how-to
+page_title: Java Obfuscation for AppBuilder applications
+slug: obfuscate-java-code
+position:
+tags: obfuscate, cordova, proguard
```

Comment:
I don't see the proguard, nor the obfuscate tags in the tags.md document. Have you forgotten to add them there?